internal/scheduler/database/sql.yaml

```diff
@@ -11,4 +11,9 @@ sql:
         sql_package: "pgx/v5"
         emit_prepared_queries: true
         emit_db_tags: true
-        emit_interface: false
\ No newline at end of file
+        emit_interface: false
+        overrides:
+          - db_type: "uuid"
+            go_type: "github.com/google/uuid.UUID"
+          - db_type: "timestamptz"
+            go_type: "time.Time"
```

Comment:
The new `sqlc` versions replaces some types with new types from the `pgx/v5/pgtype` package, some of them aren't interoperable with the codebase, specifically:
- `sqlc` now uses `pgtype.UUID` instead of `"github.com/google/uuid.UUID"`, and this breaks a lot of the codebase
- `sqlc` now uses `pgtype.Timestamptz` instead of "time.Time", this breaks the code in multiple places.

Reply:
This looks nice